packages/node-client/package.json

```diff
@@ -36,6 +36,7 @@
         "README.md"
     ],
     "devDependencies": {
+        "@nangohq/types": "^0.40.0",
```

Comment:
should we use `../types` for devDependencies?

Reply:
Since node-client is published we have to use the published version and not the local one

Reply:
does `npm install` also install the devDependencies?

Reply:
Yes, unless you pass a `--production` flag or if `NODE_ENV` environment variable is set to `production`